cadence/contracts/LostAndFound.cdc

```diff
@@ -370,6 +370,13 @@ pub contract LostAndFound {
             self.shelves <- {}
         }
 
+        pub fun ensureShelf(_ addr: Address, flowTokenRepayment: Capability<&FlowToken.Vault{FungibleToken.Receiver}>?) {
```

Comment:
could be nice if this fn returns a reference to the borrowed shelf since it seems like it usually get used after being ensured

Reply:
Sure! Definitely could help shorten a function call here and there. Seems unlikely you'll want to ensure a shelf exists and not borrow it shortly after :+1: